from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_login_window(object):
    def setupUi(self, login_window):
        login_window.setObjectName("login_window")
        login_window.resize(383, 579)
        login_window.setMinimumSize(QtCore.QSize(383, 579))
        login_window.setMaximumSize(QtCore.QSize(383, 579))
        login_window.setStyleSheet("")
        self.centralwidget = QtWidgets.QWidget(login_window)
        self.centralwidget.setObjectName("centralwidget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.centralwidget)
        self.verticalLayout.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout.setSpacing(0)
        self.verticalLayout.setObjectName("verticalLayout")
        self.login_header = QtWidgets.QFrame(self.centralwidget)
        self.login_header.setMaximumSize(QtCore.QSize(16777215, 40))
        self.login_header.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.login_header.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.login_header.setFrameShadow(QtWidgets.QFrame.Plain)
        self.login_header.setLineWidth(1)
        self.login_header.setObjectName("login_header")
        self.login_close_button = QtWidgets.QPushButton(self.login_header)
        self.login_close_button.setGeometry(QtCore.QRect(340, 10, 31, 21))
        self.login_close_button.setText("")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap(":/images/images/cross.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.login_close_button.setIcon(icon)
        self.login_close_button.setObjectName("login_close_button")
        self.login_minimize_button = QtWidgets.QPushButton(self.login_header)
        self.login_minimize_button.setGeometry(QtCore.QRect(310, 10, 31, 21))
        self.login_minimize_button.setText("")
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap(":/images/images/minimize.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.login_minimize_button.setIcon(icon1)
        self.login_minimize_button.setObjectName("login_minimize_button")
        self.verticalLayout.addWidget(self.login_header)
        self.login_body = QtWidgets.QFrame(self.centralwidget)
        self.login_body.setStyleSheet("background-color: rgb(21, 21, 21);")
        self.login_body.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.login_body.setFrameShadow(QtWidgets.QFrame.Plain)
        self.login_body.setObjectName("login_body")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.login_body)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.login_frame = QtWidgets.QFrame(self.login_body)
        self.login_frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.login_frame.setFrameShadow(QtWidgets.QFrame.Plain)
        self.login_frame.setObjectName("login_frame")
        self.tabWidget = QtWidgets.QTabWidget(self.login_frame)
        self.tabWidget.setEnabled(True)
        self.tabWidget.setGeometry(QtCore.QRect(20, 130, 321, 351))
        self.tabWidget.setStyleSheet("background-color: rgb(21, 21, 21);\n"
"color: rgb(255, 255, 255);")
        self.tabWidget.setTabPosition(QtWidgets.QTabWidget.North)
        self.tabWidget.setTabShape(QtWidgets.QTabWidget.Rounded)
        self.tabWidget.setUsesScrollButtons(True)
        self.tabWidget.setTabsClosable(False)
        self.tabWidget.setObjectName("tabWidget")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.login_username = QtWidgets.QLineEdit(self.tab)
        self.login_username.setGeometry(QtCore.QRect(70, 70, 171, 31))
        self.login_username.setStyleSheet("color: rgb(255, 255, 255); background-color: rgb(34, 34, 34);")
        self.login_username.setObjectName("login_username")
        self.login_password = QtWidgets.QLineEdit(self.tab)
        self.login_password.setGeometry(QtCore.QRect(70, 150, 171, 32))
        self.login_password.setStyleSheet("color: rgb(255, 255, 255); background-color: rgb(34, 34, 34);")
        self.login_password.setEchoMode(QtWidgets.QLineEdit.Password)
        self.login_password.setObjectName("login_password")
        self.login_button = QtWidgets.QPushButton(self.tab)
        self.login_button.setGeometry(QtCore.QRect(80, 230, 151, 41))
        font = QtGui.QFont()
        font.setFamily("Noto Serif")
        font.setPointSize(12)
        font.setBold(False)
        font.setItalic(False)
        font.setWeight(50)
        self.login_button.setFont(font)
        self.login_button.setStyleSheet("background-color: rgb(0, 173, 103);\n"
"color: rgb(0, 0, 0);")
        self.login_button.setObjectName("login_button")
        self.tabWidget.addTab(self.tab, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.signin_username = QtWidgets.QLineEdit(self.tab_2)
        self.signin_username.setGeometry(QtCore.QRect(70, 50, 171, 32))
        self.signin_username.setStyleSheet("color: rgb(255, 255, 255); background-color: rgb(34, 34, 34);")
        self.signin_username.setObjectName("signin_username")
        self.signin_password = QtWidgets.QLineEdit(self.tab_2)
        self.signin_password.setGeometry(QtCore.QRect(70, 120, 171, 32))
        self.signin_password.setStyleSheet("color: rgb(255, 255, 255); background-color: rgb(34, 34, 34);")
        self.signin_password.setEchoMode(QtWidgets.QLineEdit.Password)
        self.signin_password.setObjectName("signin_password")
        self.signin_confirm_password = QtWidgets.QLineEdit(self.tab_2)
        self.signin_confirm_password.setGeometry(QtCore.QRect(70, 160, 171, 32))
        self.signin_confirm_password.setStyleSheet("color: rgb(255, 255, 255); background-color: rgb(34, 34, 34);")
        self.signin_confirm_password.setEchoMode(QtWidgets.QLineEdit.Password)
        self.signin_confirm_password.setObjectName("signin_confirm_password")
        self.signin_button = QtWidgets.QPushButton(self.tab_2)
        self.signin_button.setGeometry(QtCore.QRect(80, 230, 151, 41))
        font = QtGui.QFont()
        font.setFamily("Noto Serif")
        font.setPointSize(12)
        self.signin_button.setFont(font)
        self.signin_button.setStyleSheet("color: rgb(0, 0, 0);\n"
"background-color: rgb(0, 173, 103);")
        self.signin_button.setObjectName("signin_button")
        self.tabWidget.addTab(self.tab_2, "")
        self.login_icon = QtWidgets.QFrame(self.login_frame)
        self.login_icon.setGeometry(QtCore.QRect(140, 0, 91, 81))
        self.login_icon.setStyleSheet("image: url(:/images/images/imago.png);")
        self.login_icon.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.login_icon.setFrameShadow(QtWidgets.QFrame.Raised)
        self.login_icon.setObjectName("login_icon")
        self.text_title = QtWidgets.QTextBrowser(self.login_frame)
        self.text_title.setGeometry(QtCore.QRect(140, 80, 111, 51))
        font = QtGui.QFont()
        font.setFamily("Noto Serif")
        font.setPointSize(18)
        font.setBold(True)
        font.setItalic(True)
        font.setWeight(75)
        self.text_title.setFont(font)
        self.text_title.setStyleSheet("color: rgb(255, 255, 255);")
        self.text_title.setFrameShape(QtWidgets.QFrame.NoFrame)
        self.text_title.setObjectName("text_title")
        self.horizontalLayout.addWidget(self.login_frame)
        self.verticalLayout.addWidget(self.login_body)
        self.login_footer = QtWidgets.QFrame(self.centralwidget)
        self.login_footer.setMaximumSize(QtCore.QSize(16777215, 30))
        self.login_footer.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.login_footer.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.login_footer.setFrameShadow(QtWidgets.QFrame.Plain)
        self.login_footer.setObjectName("login_footer")
        self.verticalLayout.addWidget(self.login_footer)
        login_window.setCentralWidget(self.centralwidget)

        self.retranslateUi(login_window)
        self.tabWidget.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(login_window)

    def retranslateUi(self, login_window):
        _translate = QtCore.QCoreApplication.translate
        login_window.setWindowTitle(_translate("login_window", "MainWindow"))
        self.login_username.setPlaceholderText(_translate("login_window", "Username"))
        self.login_password.setPlaceholderText(_translate("login_window", "Password"))
        self.login_button.setText(_translate("login_window", "Log In"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab), _translate("login_window", "Log in"))
        self.signin_username.setPlaceholderText(_translate("login_window", "Username"))
        self.signin_password.setPlaceholderText(_translate("login_window", "Password"))
        self.signin_confirm_password.setPlaceholderText(_translate("login_window", "Confirm Password"))
        self.signin_button.setText(_translate("login_window", "Sign In"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_2), _translate("login_window", "Sign in"))
        self.text_title.setHtml(_translate("login_window", "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.0//EN\" \"http://www.w3.org/TR/REC-html40/strict.dtd\">\n"
"<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
"p, li { white-space: pre-wrap; }\n"
"</style></head><body style=\" font-family:\'Noto Serif\'; font-size:18pt; font-weight:600; font-style:italic;\">\n"
"<p style=\" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;\">Imago</p></body></html>"))
import user_interface.images_rc


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    login_window = QtWidgets.QMainWindow()
    ui = Ui_login_window()
    ui.setupUi(login_window)
    login_window.show()
    sys.exit(app.exec_())
